gdc_ng_models/models/release_log.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Boolean, Column, Enum, Index, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class ReleaseLog(Base):
+
+    __tablename__ = "release_log"
+    __table_args__ = (
+        Index("release_log_project_id_idx", "program_name", "project_code"),
+    )
+
+    def __repr__(self):
+        return "<ReleaseLog(project_id={}-{}, release_number='{}', open='{}')>".format(
+            self.program_name,
+            self.project_code,
+            self.release_number,
+            self.open,
+        )
+
+    program_name = Column(Text, primary_key=True, nullable=False)
+    project_code = Column(Text, primary_key=True, nullable=False)
+    release_number = Column(Text, nullable=False)
+    data_type = Column(Enum("ssm", "cnv", name='data_type'), nullable=False)
+    is_open = Column(Boolean, nullable=False)
```

Comment:
I've seen @singergr use `controlled` and `is_genomic` too. maybe it's worth defining some properties/sets for that.
`is_genomic` will be a set inclusion check I guess

Reply:
There's a separate "genomic data" table that I think will address this concern. @philtom-ctds and I split the ticket. He's going to implement the genomic data table.

Reply:
To clarify for other reviewers, Jimmy will add the other table too.